kroxylicious/pom.xml

```diff
@@ -327,6 +327,19 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>withFilters</id>
```

Comment:
We call them `additonal-filters` in the directory structure. I think we should be consistent in the naming. 

If your not keen on the additional filters name, I'm entirely open to other suggestions.

Reply:
I renamed the profile.